python/nav/statemon/RunQueue.py

```diff
@@ -197,7 +201,7 @@ def terminate(self):
         self.stop = 1
         self.await_work.notifyAll()
         self.lock.release()
-        debug("Waiting for threads to terminate...")
+        LOGGER.info("Waiting for threads to terminate...")
```

Comment:
I've mapped 5 and 6 to info and 7 to debug. This is of course open to debate :) Mapping table in `nav.logs`.